patches/documentation+14.0.1.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/node_modules/documentation/src/parsers/parse_to_ast.js b/node_modules/documentation/src/parsers/parse_to_ast.js
+index 0eed35d..701f18d 100644
+--- a/node_modules/documentation/src/parsers/parse_to_ast.js
++++ b/node_modules/documentation/src/parsers/parse_to_ast.js
+@@ -3,7 +3,7 @@ import path from 'path';
+ 
+ const TYPESCRIPT_EXTS = {
+   '.ts': ['typescript'],
+-  '.tsx': ['typescript', 'jsx']
++  '.tsx': ['typescript', 'jsx'],
+ };
+ 
+ // this list is roughly the same as the one in prettier
+@@ -14,7 +14,7 @@ export const standardBabelParserPlugins = [
+   'classProperties',
+   'classStaticBlock',
+   'decimal',
+-  ['decorators', { decoratorsBeforeExport: false }],
++  'decorators-legacy',
```

Comment:
This is for applying the patch for 14 version,

Reply:
ya